hta/analyzers/trace_counters.py

```diff
@@ -16,7 +16,7 @@ def __init__(self):
         pass
 
     @classmethod
-    def _get_queue_length_time_series_for_rank(cls, t: "Trace", rank: int) -> pd.DataFrame:
+    def _get_queue_length_time_series_for_rank(cls, t: "Trace", rank: int) -> Optional[pd.DataFrame]:
```

Comment:
Can you please add a docstring to this function following the convention in [here](https://github.com/facebookresearch/HolisticTraceAnalysis/blob/main/hta/trace_analysis.py#L118-L141). It will help us in building a more comprehensive documentation. Thanks!